rest/replicator_test.go

```diff
@@ -3932,7 +3927,25 @@ func TestSGR2TombstoneConflictHandling(t *testing.T) {
 		},
 	}
 
+	// requireTombstone validates tombstoned revision.
+	requireTombstone := func(t *testing.T, bucket *base.TestBucket, docID string) {
+		var rawBody db.Body
+		_, err := bucket.Get(docID, &rawBody)
+		if base.TestUseXattrs() {
+			require.True(t, base.IsDocNotFoundError(err))
+			require.Len(t, rawBody, 0)
+		} else {
+			require.NoError(t, err)
+			require.Len(t, rawBody, 1)
+			rawSyncData, ok := rawBody[base.SyncXattrName].(map[string]interface{})
```

Comment:
Since we're not using xattrs, looking for base.SyncXattrName doesn't make sense here. base.SyncPropertyName is the right one to use here.

Reply:
Makes sense, this is included in the latest commit.